wdl2cwl/main.py

```diff
@@ -433,28 +433,37 @@ def get_outdir_requirement(
                 isinstance(outdir, (WDL.Expr.Apply, WDL.Expr.String))
                 and outdir.literal is None
             ):
+                # runtime["disks"] usually have strings chars like local, HDD, SDD that are not needed by CWL
```

Comment:
I don't understand how this comment relates to any of the code that surrounds it